README.md

```diff
@@ -10,4 +10,6 @@ This is the forth line on VS Code by Shiva
 
 This is the fifth line on test_branch from VSCode
 
-This is for the feature-readme-instructions branch.
\ No newline at end of file
+This is for the feature-readme-instructions branch.
+
+This is a new feature from khaliq-feature.
```

Comment:
This is where i have added the new line.

Reply:
Sounds good. Let's merge the code